articles/140_topic_and_service_name_mapping.md

```diff
@@ -265,7 +335,7 @@ Here are some examples of translations between ROS topic names and the correspon
 | `/_foo/_bar`      | Topic     | `rt___foo___bar`       |
 | `/_foo/_bar/_baz` | Topic     | `rt___foo___bar___baz` |
 
-### ROS Topic and Service Name Length Limit
+#### ROS Topic and Service Name Length Limit
```

Comment:
> i left it within the alternative section, as there is no ros prefix IMO.

I agree with @dhood, there is a topic length restriction and it's based on the partition length limit and the topic name limit in DDS. This doesn't have anything to do with "ros prefix" as I understand it.